androidApp/src/main/java/dev/johnoreilly/confetti/di/AppModule.kt

```diff
@@ -27,7 +28,9 @@ import org.koin.dsl.module
 val appModule = module {
     viewModelOf(::SessionsViewModel)
     viewModelOf(::AppViewModel)
-    viewModelOf(::ConferencesViewModel)
+    viewModel {
+        ConferencesViewModel { get<WearSettingsSync>().setConference(it) }
```

Comment:
Could `WearSettingsSync` listen to `ConfettiRepository.conferenceFlow` instead? Use that as single source of truth everywhere?

Reply:
>  Should I be making this a conference listener on ConfettiRepository?

There's already `ConfettiRepository.conferenceFlow` that you can reuse. It's the one we use for background sync, etc...

> AppSettings?

Good question, I like `ConfettiRepository` better because it doesn't leak settings. But no strong preference.

Reply:
👍 I think I wouldn't mind a global application-level coroutine scope that handles the wear <-> phone sync. At some point if it has app lifecycle, it's OK to have a coroutine running?

Reply:
I've moved it down to the Repository, and enabled via DI wiring.  I can't shake the feeling that a direct update on each change, is better than subscribing to a Flow.